src/parser/rogue/shared/resources/ComboPointTracker.js

```diff
@@ -1,11 +1,27 @@
 import RESOURCE_TYPES from 'game/RESOURCE_TYPES';
 import ResourceTracker from 'parser/shared/modules/resourcetracker/ResourceTracker';
+import SPELLS from 'common/SPELLS';
 
 class ComboPointTracker extends ResourceTracker {
   constructor(...args) {
     super(...args);
     this.resource = RESOURCE_TYPES.COMBO_POINTS;
   }
+
+  on_byPlayer_damage(event) {
+    const spellId = event.ability.guid;
+    
+    // Bonus hits from Sinister Strike are not included in the energize event, so add them in here
+    if(spellId === SPELLS.SINISTER_STRIKE_PROC.id) {      
+      let amount = 1;
+
+      if(this.selectedCombatant.hasBuff(SPELLS.BROADSIDE.id)){
```

Comment:
Again, you are not using your `RollTheBonesTracker` and its comments lead me to believe that this will sometimes be incorrect. So either `RollTheBonesTracker` has to be come a Normalizer, or your tracker needs to get used everywhere.

Reply:
As I understood .hasBuff to be working, it's essentially just keeping track of AURA_APPLIED and AURA_REMOVED events, which are still accurate for these buffs. Again I could make that part of the tracker but I just didn't want to duplicate functionality that was already available.